go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -253,7 +253,7 @@ func addDefaultConntrackRules(nodeName, gwBridge, gwIntf string) error {
 	return nil
 }
 
-func initSharedGateway(nodeName string, subnet, gwNextHop, gwIntf string,
+func initSharedGateway(nodeName string, subnet string, gwNextHop, gwIntf string,
```

Comment:
why is this changed?

Reply:
oh, I had probably changed it to take a `*net.IPNet` in an earlier version and then reverted it back, but incorrectly